docs/.vuepress/components/cdr-doc-html-example-list.vue

```diff
@@ -0,0 +1,242 @@
+<template>
+  <div class="cdr-doc-html-example-list" :class="{'cdr-doc-html-example-list--not-interactive': !interactive }">
```

Comment:
can you give me some background on why this is described as a list? it does not seem to be one in anyway that I can understand

Reply:
@mhewson did you still want to change the name to something else after our conversation yesterday? If so, any suggestions?

Reply:
It's not my fav but really not a big priority